src/ralph/backends/lrs/fs.py

```diff
@@ -50,10 +51,15 @@ def write(  # noqa: PLR0913
 
         See `FSDataBackend.write`.
         """
-        target = target if target else self.settings.DEFAULT_LRS_FILE
+        if target:
+            target = str(Path(target) / Path(self.settings.DEFAULT_LRS_FILE))
```

Comment:
Should we create the `target` directory if it doesn't exist?

Reply:
I'm not sure Ralph should be responsible for creating the directory, to stay consistent with the other backends. For the other backends, user needs to create the index, the collection or the table in advance